src/Arcus.WebApi.Security/Arcus.WebApi.Security.csproj

```diff
@@ -28,6 +28,7 @@
   <ItemGroup>
     <PackageReference Include="Arcus.Security.Providers.AzureKeyVault" Version="1.3.0" />
     <PackageReference Include="Guard.Net" Version="1.2.0" />
+    <PackageReference Include="IdentityModel" Version="3.10.10" />
```

Comment:
This is a very old version, can you update it please? Or don't we need it anymore?

Reply:
done